CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.2
+
+<!-- generated comparing v6.0.0-alpha.1..next -->
+
+_Apr 9, 2024_
+
+A big thanks to the 5 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@6.0.0-alpha.2`
+
+- &#8203;<!-- 2 -->[typescript][Select] Fix `muiName` property TypeScript error (#41726) @EyaOuenniche
```

Comment:
```suggestion
- [typescript][Select] Fix `muiName` property TypeScript error (#41726) @EyaOuenniche
```

Reply:
What's the change here? Removing the html comments ?

Reply:
> Removing the html comments?

This one :)